### PR TITLE
feat/check-process-git

### PR DESCRIPTION
**Pull request by voloshynajelena**

### CODE REVIEW

**Comment by Katia-Ch** [comment]
Comment

File: src/app/services/data.service.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ import { Injectable } from '@angular/core';
 })
 export class DataService {
   constructor(private http: HttpClient) {}
-  configUrl = 'https://api-medeor.herokuapp.com/url';
+  configUrl = 'https://api-medeor.herokuapp.com/getUser';
```
Comment (voloshyndmitry) [inline src/app/services/data.service.ts:9]: please remove it

Comment (voloshynajelena) [reply src/app/services/data.service.ts:9]: No

**Review by voloshyndmitry — CHANGES_REQUESTED** [review]
dont merge it

File: src/app/services/data.service.ts [hunk 2]
```diff
@@ -6,9 +6,9 @@ import { Injectable } from '@angular/core';
...
 
   getData(data) {
-    return this.http.get(this.configUrl, { params: { code: data } });
+    return this.http.get(this.configUrl, { params: { name: 'test', pass: 'Qweqwe34' } });
```
Comment (baronin) [inline src/app/services/data.service.ts:12]: set pass empty string

**Review by baronin — COMMENTED** [review]
test

**Pull request merged**